x-pack/legacy/plugins/maps/public/layers/styles/vector/components/symbol/vector_style_symbolize_as_editor.js

```diff
@@ -42,14 +48,15 @@ export function VectorStyleSymbolizeAsEditor({ styleProperty, handlePropertyChan
     handlePropertyChange(VECTOR_STYLES.SYMBOLIZE_AS, styleDescriptor);
   };
 
-  return (
+  const symbolizeAsForm = (
```

Comment:
This would be a good candidate for a [conditional wrapper](https://blog.hackages.io/conditionally-wrap-an-element-in-react-a8b9a47fab2), but def not critical to change